sql/modules/Roles.sql

```diff
@@ -415,7 +541,14 @@ SELECT lsmb__grant_menu('employees_manage', 49, 'allow');
 GRANT select ON employees TO public;
 
 
-SELECT lsmb__create_role('contact_edit');
+SELECT lsmb__create_role('contact_edit',
+                         $DOC$
+                         This role allows editing of existing contacts and creation of new ones.
+
+                         Combine this role with one or more 'contact_class_*' roles to be able to access
+                         contacts of the specific class. By itself, this role does not provide sufficient rights.
+                         $DOC$
+);
```

Comment:
I suggest the same wording as the one suggested for `contact_read`.

Reply:
Done.